src/unix/aix.c

```diff
@@ -937,11 +942,15 @@ int uv_fs_event_start(uv_fs_event_t* handle,
   uv__io_start(handle->loop, &handle->event_watcher, UV__POLLIN);
 
   return 0;
+#else
+  return -ENOSYS;
+#endif
 }
 
 
 int uv_fs_event_stop(uv_fs_event_t* handle) {
```

Comment:
This and _start should also return ENOSYS. At least that's how we do in SunOS, when support is not present: https://github.com/libuv/libuv/blob/v1.x/src/unix/sunos.c#L478-L502